scripts/imgtool/image.py

```diff
@@ -246,9 +246,7 @@ def save(self, path, hex_addr=None):
                     magic_size = 16
                     magic_align_size = align_up(magic_size, self.max_align)
```

Comment:
If you don't mind the unrelated changed, please remove the `magic_size` above and use `len(self.boot_magic)` in the call to `align_up` above. Otherwise the change looks good!

Reply:
Sure thing! It's in the updated PR (force pushed)